tests/Doctrine/Tests/ORM/Functional/SecondLevelCacheAbstractTest.php

```diff
@@ -224,6 +228,37 @@ protected function loadFixturesAttractionsInfo()
         $this->_em->flush();
     }
 
+    protected function loadFixturesPersonWithAddress()
```

Comment:
Shouldn't. There're more use-cases to be created around same data set (other bugs). Will keep it here to prevent me moving it by next issue I tackle.